tests/conftest.py

```diff
@@ -0,0 +1,17 @@
+"""
+Configures pytest to ignore unit tests marked as expensive unless we use the --expensive flag.
```

Comment:
There is already a conftest file in the root folder...

Reply:
Oops! Merged with this :)